non_iid_detection/non_iid_datalab.ipynb

```diff
@@ -0,0 +1,1248 @@
+{
+ "cells": [
+  {
```

Comment:
When discussing the Datapoint scores, try to mention specific Dataset Index values more, eg. "we see something interesting happening for datapoints between row 2000 and 4000..."